src/recensio/plone/profiles/default/types/Review_Article_Journal.xml

```diff
@@ -48,6 +48,7 @@
     <element value="recensio.base_review" />
     <element value="recensio.cover_picture" />
     <element value="recensio.journal_review" />
```

Comment:
@thet is there a reason why this has been added?

Reply:
FWIW in the old code this content type is based on the `JournalReviewSchema` which is a combination of `PrintedReviewSchema` and some extra fields. See also #884